core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -794,7 +794,7 @@ void core_init_mmu_regs(struct core_mmu_config *cfg)
 	 */
 	cfg->ttbcr = TTBCR_N_VALUE;
 }
-DECLARE_KEEP_PAGER(core_init_mmu_regs);
+DECLARE_KEEP_INIT(core_init_mmu_regs);
```

Comment:
I wonder if we could remove `DECLARE_KEEP_INIT()`, compare with the other implementations of this function.

Reply:
You're right. `core_init_mmu_map()` makes it an init section already. I'll update.